server/src-lib/Hasura/GraphQL/Resolve.hs

```diff
@@ -31,6 +31,7 @@ buildTx userInfo gCtx sqlCtx fld = do
                                , orderByCtx
                                , insCtxMap
                                , sqlCtx
+                               , queryRootResolver
                                ) $ case opCxt of
```

Comment:
can we change this code to be more readable? the last argument to run convert, the case matching can be separate?
there is also a typo in `opCxt`. it should be `opCtx`.